samples/first_response_file_upload.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+#
+# Exercise the opsramp module as an illustration of how to use it.
+#
+# (c) Copyright 2020-2021 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import yaml
+import logging
+import argparse
+import opsramp.binding
+
+
+def connect():
+    url = os.environ['OPSRAMP_URL']
+    key = os.environ['OPSRAMP_KEY']
+    secret = os.environ['OPSRAMP_SECRET']
+    return opsramp.binding.connect(url, key, secret)
+
+
+def parse_argv():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-d', '--debug',
+        action='store_true'
+    )
+    ns = parser.parse_args()
+    return ns
+
+
+def main():
+    ns = parse_argv()
+    if ns.debug:
+        logging.basicConfig()
+        logging.getLogger().setLevel(logging.DEBUG)
+
+    tenant_id = os.environ['OPSRAMP_TENANT_ID']
+    ormp = connect()
+    tenant = ormp.tenant(tenant_id)
+
+    payload = {'metaData': '{"category": "ALERT_FIRST_RESPONSE_TRAINING",\
+               "properties": {"inputColumns":["resource.generalInfo.resourceType"],\
+               "outputColumns":["suppressed","snoozeDuration"]}}'}

```

Comment:
Can you please put the json string into a separate variable to make it more visible. I was confused about why you needed continuation characters and it took a while to realise that payload is just a dict with one key/value and the value is a string of json. Something like this would be more obvious and easier to support:
```
jsonstr = '{"category": "ALERT_FIRST_RESPONSE_TRAINING",\
            "properties": {"inputColumns":["resource.generalInfo.resourceType"],\
            "outputColumns":["suppressed","snoozeDuration"]}}'
payload = {'metaData': jsonstr}
```

Reply:
Maybe create the dict and convert to string with json.dumps()? or comment very carefully?

Reply:
Fixed. thanks